torch/overrides.py

```diff
@@ -1789,17 +1789,17 @@ def resolve_name(f):
 
 @functools.lru_cache(None)
 def _get_tensor_methods() -> Set[Callable]:
-    """ Returns a set of the overridable methods on ``torch.Tensor`` """
+    """Return a set of the overridable methods on ``torch.Tensor``."""
     overridable_funcs = get_overridable_functions()
     methods = set(overridable_funcs[torch.Tensor])
     return methods
 
 @_disable_user_warnings
 def is_tensor_method_or_property(func: Callable) -> bool:
     """
-    Returns True if the function passed in is a handler for a
-    method or property belonging to ``torch.Tensor``, as passed
-    into ``__torch_function__``.
+    Return True if the passed-in function is a handler for a method or property.
```

Comment:
Other than `Returns`->`Return` was the phrasing a pydocstyle error? Otherwise would prefer to keep the original phrasing